src/psyclone/tests/domain/lfric/lfric_loop_test.py

```diff
@@ -88,6 +89,8 @@ def test_set_lower_bound_functions():
     a DynLoop is set to invalid values.
 
     '''
+    # Make sure we get an LFRicSymbolTable
```

Comment:
Better to use Monkeypatch for this I think. Also, as commented elsewhere, I think this suggests we need a (slight?) re-design.

Reply:
Done, and todo added.